go.mod

```diff
@@ -125,16 +124,17 @@ require (
 	github.com/vishvananda/netns v0.0.0-20171111001504-be1fbeda1936 // indirect
 	github.com/vmware/govmomi v0.20.1
 	github.com/xlab/handysort v0.0.0-20150421192137-fb3537ed64a1 // indirect
+	go.etcd.io/etcd v0.0.0-20191023171146-3cf2f69b5738
 	golang.org/x/crypto v0.0.0-20190611184440-5c40567a22f8
 	golang.org/x/lint v0.0.0-20190409202823-959b441ac422
-	golang.org/x/net v0.0.0-20190812203447-cdfb69ac37fc
+	golang.org/x/net v0.0.0-20190813141303-74dc4d7220e7
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
-	golang.org/x/sys v0.0.0-20190616124812-15dcb6c0061f
+	golang.org/x/sys v0.0.0-20190826190057-c7b8b68b1456
```

Comment:
@jpbetz do you know why etcd chose this version of `golang.org/x/sys`? it is not the go1.12 or go1.13 branch version

Reply:
I see that both golang.org/x/sys and the above golang.org/x/net were specifically added to support Risc-V in this PR https://github.com/etcd-io/etcd/pull/10834